modules/custom/az_dam/config/quickstart/editor.editor.az_standard.yml

```diff
@@ -0,0 +1,110 @@
+uuid: 2fe82e3f-9580-4c7f-9883-1697338c17a9
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.az_standard
+  module:
+    - ckeditor
+_core:
+  default_config_hash: p5wkWxOoV1gCUiMWuf-xQvDMVX2eCZhaQ_6nQwLnPYE
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.